.pre-commit-config.yaml

```diff
@@ -0,0 +1,24 @@
+repos:
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v2.3.0
+  hooks:
+    - id: check-yaml
+    - id: end-of-file-fixer
+    - id: trailing-whitespace
+- repo: https://github.com/psf/black
+  rev: 19.3b0
```

Comment:
Run `pre-commit autoupdate` and it will update black and other hooks to the latest release.

Reply:
I also started using http://pre-commit.ci/ -- it's a GitHub application that will automatically update your pre-commit configuration and will automatically push style fixes to all pull requests. 😍